DashWallet/Sources/UI/CrowdNode/New Account/NewAccountViewController.swift

```diff
@@ -80,11 +80,7 @@ final class NewAccountViewController: UIViewController, UITextViewDelegate {
 extension NewAccountViewController {
     private func configureHierarchy() {
         definesPresentationContext = true
-        view.backgroundColor = UIColor.dw_background()
-        let appearance = UINavigationBarAppearance()
-        appearance.configureWithTransparentBackground()
-        navigationItem.standardAppearance = appearance
-        navigationItem.scrollEdgeAppearance = appearance
+        view.backgroundColor = .dw_secondaryBackground()
```

Comment:
This creates issues with UI. By design, we have a white background there with a gray address box: https://www.figma.com/file/aghEG2PAqmXghOWi5BoHg5/Dash-Wallet%E3%83%BBApproved%E3%83%BBAndroid?node-id=2145%3A31181&t=FLesXM2Oin3xbgCB-0

<img src="https://user-images.githubusercontent.com/7733989/218630673-44824433-833d-4e1d-b5e6-7b1b5eddb2fb.jpeg" width="300" />

Reply:
Fixed: Ended up introducing new color for blocks like 'Primary Dash address'